kuzzleio/kuzzle
# Feat : seoUpgrade for kuzzle documentation

##  Metadata decription title and keywords update to get better google SEO. 


Adding this to most of the markdown doc headers with this type of header example :
```
title:  General Purpose Backend | Kuzzle Introduction | Guide | Core
meta:
  name: description
    content: Kuzzle main concepts and features overview
  name: keywords
    content: Kuzzle, Documentation, kuzzle write pluggins, General purpose backend
```


### How should this be manually tested?

  - Step 1 : after the merge check if the metadatas and title are updated on the documentation pages using a web browser extention
  - Step 2 : if updates aren't applied try to reindex the website using google search console
  - Step 3 : if it's good but v1 is still on top need to find new solutions (maybe around s3, google console or canonicals)

### Other changes

added the following on type: 'branch' to avoid those type of blank links : https://docs.kuzzle.io/paas-console/1/api/
```
<Redirect to="folder" />
```
